.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2025 Enactic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Lint
+on:
+  push:
+    paths:
+      - '.github/workflows/lint.yml'
+      - 'website/**'
+  pull_request:
+    paths:
+      - '.github/workflows/lint.yml'
+      - 'website/**'
+jobs:
+  typecheck:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: website
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'latest'
```

Comment:
We want to use the same Node.js as `website.yaml`:

https://github.com/enactic/openarm/blob/9dd972c3e4d98d3f994cf04a29a9f6f6537aea76/.github/workflows/website.yaml#L34

Should we use `node-version-file:` instead in both of `lint.yaml` and `website.yaml` to unify Node.js version information?